GoogleChromeLabs/ps-analysis-tool
# Chore: Integration of Tailwind CSS with Storybook

## Description


This pull request focuses on the integration of Tailwind CSS with Storybook, ensuring a seamless and consistent styling experience for our components during the development process.

## Relevant Technical Choices
- Created `postcss.config.cjs` file and includes Tailwind CSS and Autoprefixer plugins, facilitating the transformation of CSS code.
- Modified the Storybook configuration file (.storybook/main.js & .storybook/preview.js) to include the necessary loaders, plugins to handle PostCSS with Tailwind CSS.



## Testing Instructions
- Open the terminal in the root directory of the project.
- Run the command npm run storybook, and the Storybook development server will spin up.
- Open the host URL provided by Storybook in your preferred browser.
- You should see a demo component named Button displayed in the Storybook interface.
- Observe that the text color of the Button component is red. This color is a result of the Tailwind CSS class applied to the button component.
- Interact with the Button component as necessary, such as clicking or hovering over it, to ensure that the styles are responsive and behave as expected.
- Make any changes to the Tailwind CSS classes applied to the Button component in the corresponding source file (e.g., Button.js), and verify that the changes are correctly reflected in the Storybook display.


## Additional Information:



## Screenshot/Screencast
<img width="1163" alt="Screenshot 2023-07-24 at 10 59 56 AM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/108b4e62-bfe9-4a68-8932-c84162f6c5c3">



---
